ethereum_history_api/circuits/lib/src/account_int_test.nr

```diff
@@ -22,13 +22,22 @@ use dep::std::test::OracleMock;
 fn test_verify_account() {
     let _ = OracleMock::mock("get_header").returns((block_header_partial, block_header_rlp));
 
-    let _ = verify_account(account, state_proof, state_root);
+    let _ = verify_account(state_proof.key, account, state_proof, state_root);
 }
 
+#[test(should_fail)]
+fn test_verify_account_wrong_address() {
+    let _ = OracleMock::mock("get_header").returns((block_header_partial, block_header_rlp));
+
+    let _ = verify_account(state_proof_from_different_header.key, account, state_proof, state_root_from_different_header);
```

Comment:
I made a mistake, it should be state_root as the last argument, not state_root_from_different_header. I'll fix it